docs/fundamentals/application-insights.rst

```diff
@@ -0,0 +1,113 @@
+Application Insights
+====================
+By :ref:`Steve Smith <appinsights-author>` | Originally Published: 8 May 2015 
+
+Visual Studio Application Insights allows developers to insert a few lines of code into their application in order to find out how users are interacting with the app. It can also `detect and diagnose performance issues and exceptions <http://azure.microsoft.com/en-us/documentation/articles/app-insights-detect-triage-diagnose/>`_ in your applications. You can send telemetry data from web servers as well as clients/browers, as well as desktop applications and mobile devices.
```

Comment:
You can also look at http://blogs.msdn.com/b/webdev/archive/2015/05/19/application-insights-for-asp-net-5-you-re-in-control.aspx. Maybe you can borrow some text from that blog post or refer to it